src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQuerySink.java

```diff
@@ -78,9 +84,18 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
     if (config.tryGetProject() == null || config.autoServiceAccountUnavailable()) {
       return;
     }
-    // validate schema with underlying table
-    Schema schema = configuredSchema == null ? inputSchema : configuredSchema;
-    validateConfiguredSchema(schema, collector);
+
+    // Try validations which may require connectivity
+    try {
+      // Validate partition properties
+      config.validatePartitionProperties(inputSchema, configuredSchema, collector);
```

Comment:
this wasn't being called before?

Reply:
Apologies for the confusion, but validatePartitionProperties was originally private method which was called in config.validate. Since it made checks which required connectivity, I separated it out from the original config.validate method and called it manually in all spots that call config.validate.